splunklib/searchcommands/external_search_command.py

```diff
@@ -14,34 +14,31 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-from __future__ import absolute_import, division, print_function, unicode_literals
-
 from logging import getLogger
 import os
 import sys
 import traceback
-from splunklib import six
+from . import splunklib_logger as logger
+
 
 if sys.platform == 'win32':
     from signal import signal, CTRL_BREAK_EVENT, SIGBREAK, SIGINT, SIGTERM
     from subprocess import Popen
     import atexit
 
-from . import splunklib_logger as logger
+
 
 # P1 [ ] TODO: Add ExternalSearchCommand class documentation
 
 
-class ExternalSearchCommand(object):
-    """
-    """
+class ExternalSearchCommand:
     def __init__(self, path, argv=None, environ=None):
 
-        if not isinstance(path, (bytes, six.text_type)):
-            raise ValueError('Expected a string value for path, not {}'.format(repr(path)))
+        if not isinstance(path, (bytes,str)):
```

Comment:
```suggestion
        if not isinstance(path, (bytes, str)):
```

Reply:
will check for formatting in each file and commit the changes together